Sources/MeiliSearch/Model/KeyParams.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+/**
+ `KeyParams` contains all the parameters to create an API key.
+ */
+public struct KeyParams: Codable, Equatable {
+  public let description: String
+  public let actions: [String]
+  public let indexes: [String]
+  public let expiresAt: String?
+
+  public func encode(to encoder: Encoder) throws {
+    var container = encoder.container(keyedBy: CodingKeys.self)
+    try container.encode(description, forKey: .description)
```

Comment:
Can you explain this function for me (or send me some references about it), I didn't get what it will result, or how it is done (lack of swift expertise 😓)